### PR TITLE
Migrate metric to Evaluate library for tensorflow examples 

### PR DESCRIPTION
**Pull request by VijayKalmath**
# What does this PR do?
Currently, tensorflow examples use the `load_metric` function from Datasets library, commit migrates function call to `load` function to Evaluate library.

Fix for #18306

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [X] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [X] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

@sgugger

### CODE REVIEW

**Review by sgugger — COMMENTED** [review]
Those are the needed changes, thanks a lot! Do you want to apply the same to the other TensorFlow examples?

**Comment by VijayKalmath** [comment]
@sgugger, Working on the other examples. 

In contrast to pytorch examples, in the tensorflow examples, some of the different tasks  (summarization, translation token-classification ) are missing requirements.txt. 


Should I add a requirements.txt to these folders? I believe this will aid in the future when we are building tests for these examples.

**Comment by sgugger** [comment]
Yes please, that would be great!

**Comment by VijayKalmath** [comment]
@sgugger, all the examples have now been migrated. Will make PR open for review/merge

**Review by sgugger — APPROVED** [review]
Thanks a lot for working on this!
